src/sections/Emeritus-grid/Emeritus.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useStaticQuery, graphql } from "gatsby";
+import EmeritusGrid from './index'
+
+const Emeritus=props => {
+    const data=useStaticQuery(
```

Comment:
This query can directly be used in the `EmeritusGrid` file. No need for a separate file.